CHANGELOG.asciidoc

```diff
@@ -3,6 +3,95 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.8.0]]
+=== Beats version 8.8.0
+https://github.com/elastic/beats/compare/v8.7.1...v8.8.0[View commits]
+
+
+==== Bugfixes
+
+*Affecting all Beats*
+- Fix namespacing on self-monitoring {pull}32336[32336]
+- Fix race condition when stopping runners {pull}32433[32433]
+- Fix concurrent map writes when system/process code called from reporter code {pull}32491[32491]
+- Fix namespacing on self-monitoring {pull}32336[32336]
```

Comment:
These are also old PRs that were revived by merge conflicts. The newest one in this group is from July 2022 and has already been released: https://github.com/elastic/beats/pull/32491

Reply:
These can be removed entirely. Look at the PR number for an idea of how old it is. Anything that isn't at least 34xxx is suspect for this.